### DISCUSSION TITLE
Database populating "Video" folder

### DISCUSSION THREAD

**Discussion by sylgh** [gerbera/gerbera#2671] [Q&A]
Hi all,
  long time Mediatomb user, recently moved over to Gerbera when switching to a newer Debian base system.
  
  Previous on Mediatomb, I would configure the web interface to scan certain folders within my file structure to add files into the Database, which is working fine still, on my TV etc the base folder would then show "PC Directory", and then I could browse down that file structure to find the content.
  
  What I am finding in Gerbera, is that it is also adding a "Videos" folder which seems to appear randomly, when the database scans for new content, but in this "Videos" just lists all of the new videos it finds, but just into that base "Videos" folder.
  
  So Lets say ;
  > PC Directory
  >> TV
     >>> UK
     >>> US
  >> Films
     >>> DVD
     >>> Bluray
  >> Youtube
  
  Which is what I am used to under Mediatomb. But then there is the "Videos" folder, that for example, will contain a mixture of all of the above highlighted hierarchy.
  
  Mediatomb (from memory), never did this action, as I never came across it with years of usage.
  
  Does anyone know how to stop this "Videos" folder being created at random?

**Comment by sylgh** [comment]
![Capture](https://user-images.githubusercontent.com/108012100/175064533-46f48a68-1fbf-4ca4-9de9-8decc5c4e824.PNG)

Unfortunately, I stupidly deleted the "Videos" folder I discussed above before taking the screen shot. But it would be on the same base folder level as the "PC Directory" shows.

**Comment by sylgh** [comment]
![Capture](https://user-images.githubusercontent.com/108012100/175066199-b902ba49-8d2a-43c0-92cd-65250bdd9619.PNG)

Adding a single new file into "TV" under "fileserv", and rescanning, then adds the "Video" folder, as you can see in this 2nd screenshot.

What i would like to do, is stop this action of it adding the "Videos" folder, and just stick to the "PC Directory" folder, that I know and am used to.

**Reply by KarlStraussberger** [reply]
The `Video` folder is part of the virtual layout which you can turn off via configuration https://docs.gerbera.io/en/stable/config-import.html#scripting  (it's in the scripting section since 10+ years)

**Reply by sylgh** [reply]
@KarlStraussberger Thank you very much, adjusting the video section has stopped that element filling up now, and I just have the "PC Directory". Which is exactly as my older Mediatomb installation was acting.

Interesting that my old Mediatomb never acted this way, I guess thats just a difference in the prebuild debian compiled version, and the newer compiled Gerbera versions.

Thank you very much for the link. Much appreciated.
